easybuild/easyblocks/q/quantumespresso.py

```diff
@@ -54,6 +54,11 @@ def extra_options():
             'hybrid': [False, "Enable hybrid build (with OpenMP)", CUSTOM],
             'with_scalapack': [True, "Enable ScaLAPACK support", CUSTOM],
             'with_ace': [False, "Enable Adaptively Compressed Exchange support", CUSTOM],
+            'ntypx': [False, "Max number of different types of atom", CUSTOM],
+            'nsx': [False, "max number of atomic species (CP)", CUSTOM],
+            'npk': [False, "max number of k-points", CUSTOM],
+            'lmaxx': [False, "max non local angular momentum (l=0 to lmaxx)", CUSTOM],
+            'lqmax': [False, "max number of angular momenta of Q", CUSTOM],
```

Comment:
@Danzelot Please keep these lines sorted alphabetically (mixed with the ones that are already there)

Reply:
Done. Good thing to watch for.